### ISSUE TITLE
Typo in GetPostsForm, line 1265

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#485]
```
The call to __call() on line 1265 of GetPostsForm is missing the second 
parameter, and just needs a "null" or something (or else remove the second 
parameter from the function definition, since it isn't used anyway).

```

Original issue reported on code.google.com by `jonda...@gmail.com` on 6 May 2013 at 9:01

**Comment by GoogleCodeExporter** [comment]
```
Committed revision 708939.
```

Original comment by `ever...@fireproofsocks.com` on 6 May 2013 at 11:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Actually: that line applies to __set() on GetPostsQuery.  I should have looked 
at this more closely.

I've updated this line:
$this->placeholders[$p] = $this->__call($p,array());
```

Original comment by `ever...@fireproofsocks.com` on 14 May 2013 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
Sorry, what I meant was, "It isn't used within the function", not that it 
wasn't used by another caller.
```

Original comment by `jonda...@gmail.com` on 14 May 2013 at 6:56

**Comment by GoogleCodeExporter** [comment]
```
It was my bad -- I should know better than to trust my own comments haha
```

Original comment by `ever...@fireproofsocks.com` on 14 May 2013 at 3:31
- Changed state: **ReOpened**

**Comment by GoogleCodeExporter** [comment]
```
Released correction in 0.9.7.3
```

Original comment by `ever...@fireproofsocks.com` on 14 May 2013 at 7:53
- Changed state: **Fixed**
